packages/web/src/components/ui/Card/Card.tsx

```diff
@@ -1,43 +1,22 @@
 import React from 'react';
 import styles from './Card.module.scss';
 import clsx from 'clsx';
-import Chip from '../Chip/Chip';
-import ChipGroup from '../ChipGroup/ChipGroup';
-import AvatarGroup from '../AvatarGroup/AvatarGroup';
-import VulnerablePackage from '../VulnerablePackage/VulnerablePackage';
-import { Icon } from '../Icon/Icon';
 
-export type CardProps = {
-  id: string;
-  title: string;
+export type CardCommonProps = {
+  id?: string;
+  to?: string;
+  title?: string;
   icon?: string;
   description?: string;
-  packageTags?: {
-    featuredPackages: string[];
-    restPackages: number;
-  };
-  featuredSites?: {
-    iconList: string[];
-    numberOfUses: number;
-  };
-  vulnerablePackage?: {
-    name: string;
-    moreCount?: number;
-  };
-  variant?: 'toAll' | 'vulnerable';
+  variant?: 'placeholder' | 'small';
+  children?: React.ReactNode;
 };
 
-export default function Card({
-  title,
-  icon,
-  description,
-  packageTags,
-  featuredSites,
-  vulnerablePackage,
-  variant,
-}: CardProps) {
+const Card = ({ to, title, variant, icon, description, children }: CardCommonProps) => {
+  const Tag = to ? 'a' : 'div';
+
   return (
-    <div className={clsx(styles.card, variant && styles[variant])}>
+    <Tag href={to} className={clsx(styles.card, variant && styles[variant])}>
```

Comment:
We'll probably need an explicit `href={to ?? undefined}` to safeguard against possbile `false` or `''` prop values